### PR TITLE
Improve runtime regtests MP2/RPA

### PR DESCRIPTION
**Pull request by fstein93**

### CODE REVIEW

File: tests/QS/regtest-mp2-3/TEST_FILES [hunk 1]
```diff
@@ -1,3 +1,3 @@
 H2O-03.inp                                            11      3e-13            -17.156342200878413
-CH3_GPW.inp                                           11      4e-13            -7.2996767350835619
+CH3_GPW.inp                                           11      2e-5             -7.2996767350835619
```
Comment (oschuett) [inline tests/QS/regtest-mp2-3/TEST_FILES]: This is a pretty dramatic reduction in precision.

Comment (fstein93) [reply tests/QS/regtest-mp2-3/TEST_FILES]: Sorry, I was still using the -ffast-math flag. Now, it is the old precision.

**Comment by oschuett** [comment]
Thanks for this PR! Lots of progress on #174 :-)

**Comment by fstein93** [comment]
I had some time left and saw that issue.

**Pull request merged**